projects/plugins/jetpack/class.json-api-endpoints.php

```diff
@@ -1589,7 +1589,7 @@ public function get_author( $author, $show_email_and_ip = false ) {
 
 		if ( $site_id > -1 ) {
 			$author['site_ID']      = (int) $site_id;
-			$author['site_visible'] = $site_visible;
+			$author['site_visible'] = $site_visible ?? false;
```

Comment:
Any particular reason not to do this to preserve the current behavior?

```suggestion
			$author['site_visible'] = $site_visible ?? null;
```

Reply:
This value seems to be a `bool` in normal cases, so I figured I'd match that in edge cases.

Reply:
Looks like before it was always null unless `IS_WPCOM && ! $is_jetpack && $site_id > -1`. 🤷